config/features.yml

```diff
@@ -1398,6 +1398,10 @@ features:
     actor_type: user
     description: Conditionally display the "Welcome to VA" message to new (LOA1 or LOA3) users
     enable_in_development: true
+  veteran_onboarding_include_display_created_at_in_profile:
```

Comment:
There's no need to create a flipper for this feature, it's an additional field to the user serializer so no reason to have a flipper that turns it on and off